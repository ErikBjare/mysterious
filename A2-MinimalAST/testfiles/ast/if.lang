int x() {
    int a;
    a = 1;

    if(a) {
        a = 2;
    }
}
