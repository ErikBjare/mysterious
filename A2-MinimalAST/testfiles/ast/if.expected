Program
  List
    Function
      IdDecl ID="x" TYPE="int"
      List
      List
        IdStmt
          IdDecl ID="a" TYPE="int"
          Opt
        Assign
          IdUse ID="a"
          Literal LITERAL="1"
        IfStmt
          IfBlock
            IdUse ID="a"
            List
              Assign
                IdUse ID="a"
                Literal LITERAL="2"
          List
          Opt
